charts/matrix-recorder/templates/certificate.yaml

```diff
@@ -1,18 +1,13 @@
-{{ if ne .Values.environment "ci"}}
+{{ if .Values.certificate.enabled }}
 apiVersion: cert-manager.io/v1alpha2
 kind: Certificate
 metadata:
   name: matrix-recorder
+  namespace: default
 spec:
   secretName: matrix-recorder-tls
```

Comment:
We should do with the secret name the same as with the domain, define a helper and use it here and in the ingress.